src/iperf_api.c

```diff
@@ -1743,7 +1755,10 @@ get_parameters(struct iperf_test *test)
         r = -1;
     } else {
 	if (test->debug) {
-	    printf("get_parameters:\n%s\n", cJSON_Print(j));
+            char *str;
+            str = cJSON_Print(j);
+	    printf("get_parameters:\n%s\n", str );
+            free(str);
```

Comment:
suggested in order to avoid any memory leak con cJSON_Print function?

Reply:
I did a little research and the memory from `cJSON_Print()` actually does need to be explicitly freed (see the description of cJSON_Print() in cJSON's README file).  In other parts of iperf3, we use cJSON_Delete() but I think that's wrong...I think cJSON_Delete() is intended for use on JSON objects, not arbitrary strings.  I believe that cJSON_free() is intended for this use, per https://github.com/DaveGamble/cJSON/issues/5#issuecomment-298469697.  I'm going to open an issue for the use of cJSON_free() (which apparently didn't exist when iperf3 was originally written).